### ISSUE TITLE
Authentication Loop

### ISSUE THREAD

**Issue by heliodoros** [remixer-dec/YT-PMK#1]
After logging in with an account, it keeps asking me to login again

**Comment by remixer-dec** [comment]
Hi. I can't reproduce the issue on a new account, please provide more information about your case.
Do you see `script.google.com` domain in the URL bar after logging in? 
Do you have any errors in browser console? (press f12 to check)
Which browser do you use?

Have you clicked with the following sequence to log in -> Sign in with google >  Review permissions > select account > Additional settings > Go to page ?

**Comment by heliodoros** [comment]
> Hi. I can't reproduce the issue on a new account, please provide more information about your case. Do you see `script.google.com` domain in the URL bar after logging in? Do you have any errors in browser console? (press f12 to check) Which browser do you use?
> 
> Have you clicked with the following sequence to log in -> Sign in with google > Review permissions > select account > Additional settings > Go to page ?

Thank you for taking the time to look into my issue,

I do see the script.google.com domain after logging in. 
I am getting the following errors in the console 
`Cookie “_ga_devsite” does not have a proper “SameSite” attribute value. Soon, cookies without the “SameSite” attribute or with an invalid value will be treated as “Lax”. This means that the cookie will no longer be sent in third-party contexts. If your application depends on this cookie being available in such contexts, please add the “SameSite=No`

and 

`
Cookie “SEARCH_SAMESITE” with the “SameSite” attribute value “Lax” or “Strict” was omitted because of a cross-site redirect.ne“ attribute to it.
`

I have tried with Firefox, Chrome, Chrome on android and Edge
Basically as soon as I click allow, with any account, I am redirected to the Review Permissions page.

I am attaching screenshots in the order of events ss1, ss2, ss3 and ss4 which resemble the flow I followed
![ss1](https://user-images.githubusercontent.com/17793315/183725943-6e3435ed-4fa4-41b8-92ce-9b128788d6a2.png)
![ss2](https://user-images.githubusercontent.com/17793315/183725951-d9ad55a0-a682-4712-ab79-44caed1d2e8a.png)
![ss3](https://user-images.githubusercontent.com/17793315/183725958-4ac2c6dc-a1b8-47b6-b114-33e8c7ca3aed.png)
![ss4](https://user-images.githubusercontent.com/17793315/183725963-398b5236-a5f0-4156-98c9-ad11b192a91c.png)
.

**Comment by remixer-dec** [comment]
Honestly, I have no idea why this is happening, especially on multiple accounts/devices/browsers. 
Can you see the app listed here https://myaccount.google.com/permissions ?

**Comment by heliodoros** [comment]
> Honestly, I have no idea why this is happening, especially on multiple accounts/devices/browsers. 
> Can you see the app listed here https://myaccount.google.com/permissions ?
> 
> 

The App does not appear in the permissions list. It's the first time I have encountered something similar too.

**Comment by heliodoros** [comment]
I've tried everything. I even made a copy of the App and messed around with it but I had no good results. It keeps asking me to Review Permissions and the App is not registering in the Allowed Apps in the Google Permissions page. I tried from multiple devices and with different accounts. It's a shame because this App is exactly what I need right now as the YouTube algorithm is deleting my 15 year old playlists for no reason. It has been happening to many people. I also find the way you developed this app to be very interesting and straight forward.

**Comment by heliodoros** [comment]
After some intense Googling, I found this similar thread [https://community.smartsheet.com/discussion/19876/smartsheet-merge-permissions-issue](url) , but the solution is something I've already tried and did not work for me. Maybe it's related to accounts that contain more accounts.

**Comment by remixer-dec** [comment]
Thank you for your feedback! Google has an [issue tracker](https://issuetracker.google.com/issues), where you can try to describe the issue, however it is likely that they will ask you to provide a more simple example, or say that they cannot reproduce it, so it's better to specify that only your account(s) are affected by this.
I can also try to submit a [verification application](https://developers.google.com/apps-script/guides/client-verification) but there is a chance that it gets rejected and I don't really understand the necessity of creation of a separate GCP project only for that purpose. Anyways, it requires some time, so i'll be able to try it later this month.
